src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/CCJStaffNotificationService.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.CCJRequestSubmittedNotificationEmailContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.DefaultJudgementContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.EmailContent;
+import uk.gov.hmcts.cmc.claimstore.utils.PartyUtils;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+import uk.gov.hmcts.cmc.email.EmailService;
+import uk.gov.hmcts.reform.cmc.pdf.service.client.PDFServiceClient;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static java.util.Collections.singletonList;
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.email.EmailAttachment.pdf;
+
+@Service
+public class CCJStaffNotificationService {
+
+    public static final String FILE_NAME_FORMAT = "%s-default-judgement.pdf";
+
+    private final EmailService emailService;
+    private final StaffEmailProperties staffEmailProperties;
+    private final PDFServiceClient pdfServiceClient;
+    private final CCJRequestSubmittedNotificationEmailContentProvider ccjRequestSubmittedEmailContentProvider;
+    private final DefaultJudgementContentProvider defaultJudgementContentProvider;
+
+    @Autowired
+    public CCJStaffNotificationService(
```

Comment:
why CCJ here and CountyCourtJudgment for the staff notification handler

Reply:
it's check style which enforces the word length to particular length. Event is names as CountyCourtJudgmentSubmittedEvent, do you think we should change that to CCJSubmittedEvent?

Reply:
that's fine I think, just asking for a good reason.